Assets/BossRoom/Scripts/Shared/ProfileManager.cs

```diff
@@ -0,0 +1,44 @@
+using UnityEngine;
+
+#if UNITY_EDITOR
+using ParrelSync;
+#endif
+
+public static class ProfileManager
```

Comment:
Namespace 👀

Reply:
Oops, I really should develop a reflex for those